SETUP.md

```
@@ -0,0 +1,179 @@
+This document outlines the setup process for the MidiStems project, which uses Flutter for the frontend and Rust for audio processing via FFI.
+
+## Prerequisites
+
+- Rust (1.87.0 or later)
+- Flutter (3.32.1 or later)
+- ALSA development libraries (for Linux)
+
+## Rust Setup
+
+1. Install Rust using rustup:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)